content/docs/quickstart/azure/modify-program.md

```diff
@@ -19,91 +21,60 @@ const pulumi = require("@pulumi/pulumi");
 const azure = require("@pulumi/azure");
 
 // Create an Azure Resource Group
-const resourceGroup = new azure.core.ResourceGroup("resourceGroup", {
-    location: "WestUS",
-});
+const resourceGroup = new azure.core.ResourceGroup("resourceGroup");
 
-// Create an Azure Container Group
-const container = new azure.containerservice.Group("nginx", {
-    containers: [{
-        name: "nginx",
-        image: "nginx",
-        memory: 1,
-        cpu: 1,
-        ports: [{
-            port: 80,
-            protocol: "TCP"
-        }],
-    }],
-    osType: "Linux",
+// Create an Azure resource (Storage Account)
+const account = new azure.storage.Account("storage", {
     resourceGroupName: resourceGroup.name,
-    location: resourceGroup.location,
+    accountTier: "Standard",
+    accountReplicationType: "LRS",
+    enableBlobEncryption: true,
```

Comment:
The reason I didn't keep this similar to AWS is because creating a KeyVault and adding secrets to it in Azure involves a few extra steps and likely to be a bit more advanced for someone trying to do a "quickstart", so I chose to use the Azure managed encryption and another attribute that enforces HTTPS traffic only.

Reply:
Sounds good.

Reply:
So it turned out that the `enableBlobEncryption` is `true` by default, which is good, but that meant I had to remove it from here. Instead, now I am using the HTTPS-only attribute as the update.